### ISSUE TITLE
Add contribution metrics beyond commit count

### ISSUE THREAD

**Issue by cuviper** [rust-lang-nursery/thanks#47]
There are many ways to contribute that don't show up in commit counts.  It would be nice if these were acknowledged too, although I don't know how well these can be tracked.  Some ideas:

- Filing issues
- Opening pull requests (whether or not they ultimately get merged)
- Commenting on issues and pull requests
- Participating in discussions on either the users or internals forums
- Chatting in logged IRC channels (this may be too noisy)
- Third-party participation?  stack overflow, reddit, etc.

**Comment by steveklabnik** [comment]
Yes, this would be cool. It requires a lot more API integration than we currently have, so it'd be a big step, but I'd like to see it as well.
